.github/dependabot.yml

```diff
@@ -1,12 +0,0 @@
-version: 2
-updates:
-  - package-ecosystem: github-actions
-    directory: '/'
-    schedule:
-      interval: weekly
-    open-pull-requests-limit: 10
-  - package-ecosystem: npm
```

Comment:
Can we only remove the `npm` one and keep the `github-actions` one?

Reply:
Shure.

Reply:
I'd rather just remove. We can always add back by checking git history.

Reply:
Ok, lets go to remove.

Reply:
@AugustinMauroy I'm still waiting.